aws-janitor/resources/cloud_formation_stacks.go

```diff
@@ -87,12 +96,14 @@ func (CloudFormationStacks) ListAll(opts Options) (*Set, error) {
 }
 
 type cloudFormationStack struct {
-	id   string
-	name string
+	account string
+	region  string
+	id      string
+	name    string
 }
 
 func (p cloudFormationStack) ARN() string {
-	return p.id
+	return fmt.Sprintf("arn:aws:cloud-formation:%s:%s:stack/%s", p.region, p.account, p.id)
```

Comment:
nit: it seems like most of the real ARNs don't use dashes in the service name (i.e. make this `cloudformation`, not `cloud-formation`)